File: Backend_DotNet/WheelShare_dotNet/WheelShare_dotNet/DTOs/EstimateFareDto.cs
using System.ComponentModel.DataAnnotations;

namespace WheelShare_dotNet.DTOs
{
    public class EstimateFareDto
    {
        [Required]
        public double SourceLat { get; set; }

        [Required]
        public double SourceLng { get; set; }

        public string SourceAddress { get; set; }

        [Required]
        public double DestinationLat { get; set; }

        [Required]
        public double DestinationLng { get; set; }

        public string DestinationAddress { get; set; }
    }
}

File: Backend_Springboot/cab-booking/src/main/java/com/wheelshare/cabbooking/dto/EstimateFareDto.java
package com.wheelshare.cabbooking.dto;

import lombok.Data;

@Data
public class EstimateFareDto {
    private Double sourceLat;
    private Double sourceLng;
    private String sourceAddress;
    private Double destinationLat;
    private Double destinationLng;
    private String destinationAddress;
}
